src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -447,7 +447,7 @@ public function getProtocolVersion()
      * If the status text is null it will be automatically populated for the known
      * status codes and left empty otherwise.
      *
-     * @return Response
+     * @return $this
```

Comment:
`$this` is allowed

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#appendix-a-types

> keyword          = "resource" / "self" / "static" / "string" / "true" / "void" / "$this"

For PHPStorm `$this` gives way better DX so it seems (opposed to `self`).